### PR TITLE
Odyssey of the Dragonlords initial

### PR DESCRIPTION
**Pull request by Nevofix**
Content from Odyssey of the Dragonlords, Arcanum Worlds (request #431 )

Changed existing files:
- third-party.index

New files:
- arcanum-worlds.index
- odyssey-of-the-dragonlords.index
- source.xml

- aw-ootd-spells.xml
- aw-ootd-weapons.xml
- race-thyleancentaur.xml
- race-thyleanmedusa.xml
- race-thyleanminotaur.xml
- race-thyleannymph.xml
- race-thyleansatyr.xml
- race-thyleansiren.xml
- subclass-barbarian-herculean.xml
- subclass-bard-epicpoetry.xml
- subclass-cleric-prophecy.xml
- subclass-druid-sacrifice.xml
- subclass-fighter-hoplite.xml
- subclass-monk-shield.xml
- subclass-paladin-dragonlord.xml
- subclass-ranger-amazonian.xml
- subclass-rogue-odyssean.xml
- subclass-sorcerer-demigod.xml
- subclass-warlock-fates.xml
- subclass-wizard-academy.xml

### CODE REVIEW

**Comment by NickVendel** [comment]
Yeah, some things, i would like to mention.
1. You usually want to reference request via `#[number]` (like #431 in this case)
2. Next time you should create different branch for new content, your `master` branch should be the same as `aurorabuilder:master`, so to easily revert and start over without copying/doing over whole repository in case something goes wrong.
3. In description of the pull it's probably more useful to mention what exactly did you added, just so people don't need to go through files to get what exactly did you added. Though it's not a requirement.

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleanmedusa.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    </description>
+    <sheet>
+      <description>You can attack with your snake hair. This is a melee weapon attack with an attack bonus equal to +{{snake hair:bonus}}. It does 1d6 piercing damage on a hit, and your target must make a DC 12 Constitution saving throw or else they are poisoned until the beginning of your next turn.</description>
+    </sheet>
+    <rules>
+      <stat name="snake hair:bonus" value="0" />
+      <stat name="snake hair:bonus" value="proficiency" />
+      <stat name="snake hair:bonus" value="dexterity:modifier" />
+    </rules>
+  </element>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleanmedusa.xml]: the `<stat name="snake hair:bonus" value="0" />` can be left out

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/aw-ootd-weapons.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		<setters>
+			<set name="category">Weapons</set>
+			<set name="cost" currency="gp">10</set>
+			<set name="weight" lb="2">2 lb.</set>
+			<set name="slot">onehand</set>
+            <set name="range">60/120</set>
+			<set name="damage" type="slashing">1d6</set>
+			<set name="proficiency">ID_PROFICIENCY_WEAPON_PROFICIENCY_CHAKRAM</set>
+		</setters>
+	</element>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/aw-ootd-weapons.xml]: when adding new weapons you'll also have to create a proficiency element for it and include the proficiency in the correct 'parent' proficiency (e.g. simple/martial)

(example: [Double-Bladed Scimitar](https://github.com/aurorabuilder/elements/blob/master/supplements/eberron-rising-from-the-last-war/items.xml#L674) from Eberron: Rising from the Last War)

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/aw-ootd-weapons.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<author url="https://www.arcanumworlds.com/">Arcanum Worlds</author>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/aw-ootd-weapons.xml]: `<author url="https://www.arcanumworlds.com/">Arcanum Worlds</author>`

these lines at the top can be left out, they are not used anymore

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleanminotaur.xml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        <span class="feature">Labyrinthine Vision.</span>Your eyes are adapted to the dark conditions of deep canyons and underground labyrinths, giving you darkvision out to 60 ft. You have advantage on skill checks made to solve maze-like puzzles. Additionally, you automatically succeed on saving throws against <i>maze</i> and <i>hypnotic pattern</i>.<br/>
+        <span class="feature">Colorblindness.</span>You see the world in shades of red and grey, leaving you incapable of discerning any color except for very bright reds.<br/>
+        <span class="feature">Cursed Transformation.</span>Starting at 5th level, you may use your bonus action to transform yourself into a <b>bull</b> using the rules of the <i>polymorph</i> spell (no concentration required). This ability recharges after a long rest. This ability automatically triggers if you suffer prolonged exposure to very bright shades of red. Starting at 9th level, this ability transforms you into a <b>dire bull</b>.<br/>
+      </p>
+    </description>
+    <sheet display="false" />
+    <setters>
+
+    </setters>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleanminotaur.xml]: you can add the minotaur names

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleannymph.xml [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+        <span class="feature">Speed.</span>Your base walking speed is 30 feet.<br/>
+        <span class="feature">Language.</span>You can speak, read, and write Common and Sylvan.<br/>
+        <span class="feature">Enthralling Beauty.</span>You possess unearthly grace and beauty, which allows you to dazzle and charm those who are susceptible to such things. You have proficiency in the Persuasion skill. Additionally, you can cast the <i>charm person</i> spell once with this trait and regain the ability to do so when you finish a short or long rest. Charisma is your spellcasting ability for this spell.<br/>
+        <span class="feature">Nymph Ancestry.</span>When you create your nymph, choose one of the following subraces: Aurae, Dryad, Naiad, Nereid, or Oread. You gain access to special traits and magic depending upon your ancestry.
+      </p>
+    </description>
+    <sheet display="false" />
+    <setters>
+
+    </setters>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleannymph.xml]: you can add the nymph names

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleannymph.xml [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+      <description level="7">
+        You can breathe underwater, and you have a swimming speed of 40 ft.<br/>
+        1/short rest. <i>Fog cloud</i>, and <i>Water walk</i>.
+      </description>
+    </sheet>
+    <rules>
+      <grant type="Spell" id="ID_PHB_SPELL_FOG_CLOUD" level="3" />
+      <grant type="Spell" id="ID_PHB_SPELL_WATER_WALK" level="7" />
+    </rules>
+  </element>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleannymph.xml]: the racial swimming speed can be added using an `innate speed:swim` stat

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleansatyr.xml [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+        <span class="feature">Language.</span>You can speak, read, and write Common and Sylvan.<br/>
+        <span class="feature">Fey Heritage.</span>You have advantage on saving throws against being charmed or frightened, and spells can’t put you to sleep.<br/>
+        <span class="feature">Memory for Music.</span>You have proficiency with one instrument of your choice. You have advantage on Performance checks made with the selected instrument. You can also memorize and perform any song after hearing it only once.<br/>
+        <span  class="feature">Enchanting Music.</span>You can cast the <i>minor illusion</i> cantrip. When you reach 3rd level, you can cast the <i>sleep</i> spell once with this trait and regain the ability to do so when you finish a long rest. When you reach 5th level, you can also cast the <i>suggestion</i> spell with this trait and regain the ability to do so when you finish a long rest. Casting these spells requires access to an instrument that you are proficient with. Charisma is your spellcasting ability for these spells.
+      </p>
+    </description>
+    <sheet display="false" />
+    <setters>
+
+    </setters>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleansatyr.xml]: names can be added

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleansiren.xml [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+        <span class="feature">Flight.</span>You have a flying speed of 30 feet. To use this speed, you can’t be wearing medium or heavy armor.<br/>
+        <span class="feature">Songs of Sorrow.</span>Your lamentations have a powerful effect on anyone who can hear them. You can cast the <i>charm person</i> spell once with this trait and regain the ability to do so when you finish a short rest. When you reach 3rd level, you can cast the <i>enthrall</i> spell once with this trait and regain the ability to do so when you finish a short rest. When you reach 5th level, you can cast the <i>hold person</i> spell once with this trait and regain the ability to do so when you finish a short rest. Charisma is your spellcasting ability for these spells, and the targets of your spells must have the ability to hear you singing.
+      </p>
+    </description>
+    <sheet display="false" />
+    <setters>
+
+    </setters>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleansiren.xml]: siren names can be added

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleansiren.xml [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+  </element>
+
+  <element name="Flight" type="Racial Trait" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_RACIAL_TRAIT_THYLEAN_SIREN_FLIGHT">
+    <description>
+      <p>You have a flying speed of 30 feet. To use this speed, you can’t be wearing medium or heavy armor.</p>
+    </description>
+    <sheet>
+      <description>You have a flying speed of 30 feet. To use this speed, you can’t be wearing medium or heavy armor.</description>
+    </sheet>
+  </element>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleansiren.xml]: innate fly speed statistic can be added (a statistic rule can have equipped requirements for wearing armor, see [aarakocra ](https://github.com/aurorabuilder/elements/blob/master/supplements/princes-of-the-apocalypse/races/race-aarakocra.xml#L77-L87)for example)

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-cleric-prophecy.xml [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_MINDFUL_SENSES" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_BLESSING_OF_FORESIGHT" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_CD_PROPHETIC_TRANCE" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_VISIONS_OF_DANGER" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_HEALING_VAPORS" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_PERFECT_FORESIGHT" />
+    </description>
+    <sheet>
+      <description>The gods of the forge are patrons of artisans who work with metal, from a humble blacksmith who keeps a village in horseshoes and plow blades to the mighty elf artisan whose diamond-tipped arrows of mithral have felled demon lords.</description>
+    </sheet>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-cleric-prophecy.xml]: some copy/paste left over from another archetype

all archetype sheet descriptions can be set to `display="false"` as they will no longer be shown in the features section to save space

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-paladin-dragonlord.xml [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_OATH_OF_THE_DRAGONLORD_OATH_SPELLS" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_OATH_OF_THE_DRAGONLORD_PSEUDODRAGON_FAMILIAR" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_OATH_OF_THE_DRAGONLORD_CHANNEL_DIVINITY" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_OATH_OF_THE_DRAGONLORD_DRAGONLORDS_BOND" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_OATH_OF_THE_DRAGONLORD_YOUNG_DRAGON" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_OATH_OF_THE_DRAGONLORD_UNBREAKABLE_BOND" />
+    </description>
+    <sheet>
+      <description>The Oath of Conquest sets a paladin on a difficult path, one that requires a holy warrior to use violence only as a last resort.</description>
+    </sheet>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-paladin-dragonlord.xml]: incorrect description

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-ranger-amazonian.xml [hunk 12]
```diff
@@ -0,0 +1,256 @@
...
+			<set name="constitution">10</set>
+			<set name="intelligence">6</set>
+			<set name="wisdom">8</set>
+			<set name="charisma">7</set>
+            <set name="ac">{{companion:ac}} (natural armor)</set>
+			<set name="hp">{{companion:hp:max}}</set>
+			<set name="speed">10 ft., fly 80 ft.</set>
+			<set name="immunities">fire, poison, psychic</set>
+			<set name="conditionImmunities">charmed, poisoned</set>
+			<set name="saves">Str +{{companion:save}}, Dex +{{companion:save}}, Con +{{companion:save}}, Int +{{companion:intsave}}, Wis +{{companion:wissave}}, Cha +{{companion:chasave}}</set>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-ranger-amazonian.xml]: `{{companion:ac}}` are not supported here, just the actual value

same for hp/saves etc

File: third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-wizard-academy.xml [hunk 13]
```diff
@@ -0,0 +1,162 @@
...
+		<description>
+			<p>Academy philosophers are great thinkers who do not specialize in any particular spell school, but rather spend their time analyzing the nature of reality and applying that learning to their magic. They usually spend many years studying at venerable schools where philosophers from many different disciplines gather. They enjoy participating in endless debates and dialogues, and they always seek to score points on those who champion rival schools of thought.</p>
+			<div element="ID_AW_OOTD_ARCHETYPE_FEATURE_ACADEMY_PHILOSOPHICAL_SCHOOL" />
+			<div element="ID_AW_OOTD_ARCHETYPE_FEATURE_ACADEMY_MATHEMATICAL_PRINCIPLES" />
+			<div element="ID_AW_OOTD_ARCHETYPE_FEATURE_ACADEMY_METAPHYSICAL_PRINCIPLES" />
+			<div element="ID_AW_OOTD_ARCHETYPE_FEATURE_ACADEMY_MAGICAL_PARADOX" />
+		</description>
+		<sheet>
+			<description>War mages act fast in battle, using their spells to seize tactical control of a situation.</description>
+		</sheet>
```
Comment (swdriessen) [inline third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-wizard-academy.xml]: incorrect sheet description

**Comment by Nevofix** [comment]
I performed your requested changes. Might want to check if I did the proficiency issue right. I can add the treasures later, but I can probably only start on that later next week. Also, if you merge this, I can make a new branch in my fork so I'm no longer on my master branch.

**Comment by swdriessen** [comment]
Thanks, you can view the commits to see the changes i made. (mostly cleanups) You can create the treasures when you have time, no rush.

You can clean your master branch or just delete the fork and fork the repo again to do it the quick and easy way to get a fresh start.

**Pull request merged**